### ISSUE TITLE
Navbar contrast ratio

### ISSUE THREAD

**Issue by dangerdak** [FAC-11/week1-BAJA#62]
Text should have a contrast ratio of at least 4.5

https://github.com/GoogleChrome/accessibility-developer-tools/wiki/Audit-Rules#ax_color_01

In developer tools, under the 'elements' tab if you look at accessibility properties while the element is selected, it will recommend a similar colour which provides high enough contrast and let you easily try out the recommendation in chrome.

This only comes up as an issue when javascript is enabled! Can't see why :(
